pkg/ckecli/cmd/vault_init.go

```diff
@@ -59,6 +60,30 @@ path "cke/*"
 }`
 )
 
+func readPasswordFromStdTerminal(prompt string) (string, error) {
+	if !term.IsTerminal(int(os.Stdin.Fd())) || !term.IsTerminal(int(os.Stdout.Fd())) {
+		return "", fmt.Errorf("stdin and stdout are not terminals")
+	}
+
+	screen := struct {
+		io.Reader
+		io.Writer
+	}{os.Stdin, os.Stdout}
+
+	oldState, err := term.MakeRaw(int(os.Stdin.Fd()))
+	if err != nil {
+		return "", err
+	}
+	defer term.Restore(0, oldState)
+
+	t := term.NewTerminal(screen, "")
+	p, err := t.ReadPassword(prompt)
```

Comment:
A new terminal does not seem necessary.
I'm not sure whether `MakeRaw()` is necessary or not.

```suggestion
	fmt.Print(prompt)
	p, err := term.ReadPassword(int(os.Stdin.Fd()))
```

Reply:
according to my local poc just now, we have to print '\n' after ReadPassword. i.e. the following is enough:
```
	fmt.Print(prompt)
	p, err := term.ReadPassword(int(os.Stdin.Fd()))
	fmt.Println()
```

`MakeRaw` is required if we use `t.ReadPassword` but it does not seems required if we use `term.ReadPassword`.